dlt/cli/magics.py

```diff
@@ -0,0 +1,286 @@
+
+
+from dlt.common.runtime.exec_info import is_notebook, is_ipython
+from dlt.cli import echo as fmt, echo
+
+import typing as t  # type: ignore
+from hyperscript import h  # type: ignore
```

Comment:
does hyperscript comes with iPython or people need to install this dependency to use magics? if the second one better you rewrite the html generating code

also this lib looks rather obscure and was not recently updated...